dashboard/app/controllers/reports_controller.rb

```diff
@@ -56,6 +56,23 @@ def all_usage
     render 'usage', formats: [:html]
   end
 
+  def search_for_teachers
+    authorize! :read, :reports
+
+    if (params[:emailFilter])
```

Comment:
in ruby nil (and of course, empty string) both print nothing when used in "#{}":

````
irb(main):001:0> whatev = nil
=> nil
irb(main):002:0> "%#{whatev}%"
=> "%%"
irb(main):003:0> whatev = ''
=> ""
irb(main):004:0> "%#{whatev}%"
=> "%%"
irb(main):005:0> whatev = 'something'
=> "something"
irb(main):006:0> "%#{whatev}%"
=> "%something%"
````
so you could take advantage of this by doing just `email_filter = "%#{params[:emailFilter]}%"`

also we usually don't use parens when not necessary/useful for grouping/precedence (for the ruby interpreter or human readers) so you can leave the () out of simple ifs like this if you want.